src/ir.cpp

```diff
@@ -1307,6 +1307,7 @@ static IrInstruction *ir_build_un_op(IrBuilder *irb, Scope *scope, AstNode *sour
     IrInstructionUnOp *br_instruction = ir_build_instruction<IrInstructionUnOp>(irb, scope, source_node);
     br_instruction->op_id = op_id;
     br_instruction->value = value;
+    br_instruction->lval = LValNone;
```

Comment:
This was probably my mistake, but can you rename `br_instruction` to either `instruction` or `un_op_instruction`?